package.json

```diff
@@ -24,7 +24,7 @@
     "launchdarkly-js-client-sdk": "^2.18.3",
     "lodash": "^4.17.20",
     "remove-accents": "^0.4.2",
-    "sbc-common-components": "2.4.8",
+    "sbc-common-components": "2.4.14",
```

Comment:
Please update the NR app version on line 3 as well (so we can easily tell what's deployed).

Reply:
done

Reply:
Thanks.
Please also check in the updated package-lock.json.

Reply:
Yes